packages/compiler/init/init.ts

```diff
@@ -339,13 +339,20 @@ async function writeFiles(host: CompilerHost, config: ScaffoldingConfig) {
     return;
   }
   for (const file of config.files) {
-    await writeFile(host, config, file);
+    if (file.path === false) {
+      await host.rm(file.destination);
```

Comment:
yeah this feels really hacky, it should skip the generation of those not create then delete